core/node/core.go

```diff
@@ -51,15 +51,13 @@ func Pinning(bstore blockstore.Blockstore, ds format.DAGService, repo repo.Repo)
 		return rootDS.Sync(filestore.FilestorePrefix)
 	}
 	syncDs := &syncDagService{ds, syncFn}
-	syncInternalDag := &syncDagService{internalDag, syncFn}
 
-	pinning, err := pin.LoadPinner(rootDS, syncDs, syncInternalDag)
+	ctx, cancel := context.WithTimeout(context.TODO(), 2*time.Minute)
```

Comment:
I grabbed the 2 minutes from the other PR, but ideally rebuilding shouldn't take this long.